templates/birds/_list.html.j2

```diff
@@ -1,28 +1,26 @@
 {% import 'utility.macros.j2' as utility %}
 {% import "birds/macros.j2" as birds %}
 
+{% set all_posts = get_recent_blog_entries(1000) %}
 
-{% if config.birds %}
-  {% set all_posts = get_recent_blog_entries(1000) %}
+{% if module.show %}
+  {% for author in module.show %}
+    {% set page = all_pages|filter_pages('bird', 'eq', author)|first %}
+    {# @@@ Moving OSS data into pages will help clean this up... #}
+    {% set oss = config.oss_by_author[author] %}
+    {# @@@ Adding global access to posts-by-author could clean this up... #}
+    {% set posts = all_posts|selectattr('author', 'equalto', author)|list %}
 
-  {% for bird in config.birds %}
-    {% set posts = all_posts|selectattr('author', 'equalto', bird.author)|list %}
-    {% set oss = config.oss_by_author[bird.author] %}
-
-    {{ birds.section(bird, oss, posts) }}
+    {{ birds.section(page, oss, posts) }}
   {% endfor %}
-{% endif %}
+{% else %}
```

Comment:
Added comments in 6b56f29 — you can either declare the birds to show (`module.show`) or it will show all pages with the `bird` attribute